content/events/hacktoberfest.adoc

```diff
@@ -14,56 +14,52 @@ opengraph:
   image: /images/hacktoberfest/hacktoberfest_2022_social.png
 ---
 
-image:/images/hacktoberfest/hacktoberfest_2022.svg[Hacktoberfest, role=center]
+image:/images/hacktoberfest/hacktoberfest_2023_logo.png[Hacktoberfest, role=center]
 
 link:https://hacktoberfest.com/[Hacktoberfest]
-is a month long celebration of open source software.
+is a month-long celebration of open source software.
 It happens every year in October.
-During this event everyone can support open-source by contributing changes, and then earn a limited edition swag.
-
-We invite everyone to participate and to contribute to the Jenkins project,
-regardless of their background and Jenkins experience.
-There are many ways to
-link:/participate/[contribute] to Jenkins during Hacktoberfest.
-You can work on code or documentation,
-contribute to link:https://wiki.jenkins.io/display/JENKINS/Internationalization[localization],
-create new link:/artwork[artwork], etc.
-You can also share your experiences with Jenkins by link:https://github.com/jenkins-infra/jenkins.io/blob/master/CONTRIBUTING.adoc#adding-a-blog-post[creating a new blogpost].
-Generally, any pull requests in GitHub would qualify.
-See the link:/participate/[Contribute and Participate] page for more information about how to contribute.
+During this event everyone can support open-source by contributing changes, and then earn a unique digital reward.
+This year marks the 10th anniversary of Hacktoberfest.
 
+We invite everyone to participate and to contribute to the Jenkins project, regardless of their background and Jenkins experience.
+There are many ways to link:/participate/[contribute] to Jenkins during Hacktoberfest.
+You can work on code or documentation, contribute to link:https://wiki.jenkins.io/display/JENKINS/Internationalization[localization], or create new link:/artwork[artwork], etc.
+You can also share your experiences with Jenkins by link:https://github.com/jenkins-infra/jenkins.io/blob/master/CONTRIBUTING.adoc#adding-a-blog-post[creating a new blogpost].
+Generally, any four non-spammy pull requests in GitHub or merge requests in GitLab would qualify.
+Please see the link:/participate/[Contribute and Participate] page for more information about how to contribute.
 
 == Quick start
 
 1. Sign-up to Hacktoberfest on link:https://hacktoberfest.com[the event website].
-2. Join link:https://app.gitter.im/#/room/#jenkinsci_hacktoberfest:gitter.im[our Gitter channel].
-3. Everything is set, just start creating pull-requests!
-** If a repository has no `hacktoberfest` topic set, contact the maintainers to see if they are willing to accept a contribution.
-
+2. Join link:https://matrix.to/#/#jenkinsci_hacktoberfest:gitter.im[our Gitter channel].
```

Comment:
Why was the link changed from a ready-to-use service to the matrix URL shortener?

Reply:
The change was intentional so as to provide a more modern platform / medium for people to connect with us, as we have received complaints about the Gitter links before.

Reply:
> was intentional so as to provide a more modern platform / medium for people to connect with us

![Screenshot 2023-09-13 at 17 02 37](https://github.com/jenkins-infra/jenkins.io/assets/13383509/b9f3935a-7764-4066-ab0f-90106382356e)

I don't see any way to connect with anyone here, that's just a URL shortener when the former already opens a web client with a chat you can start talking in.

>  as we have received complaints about the Gitter links

I'm interested in what the issues are, which would be addressed by replacing the chat window with a URL shortener pointing to a generic destination.

Reply:
@krisstern & @NotMyFault 
I propose to keep the link as it was before. This will give us a little more time to debate the policy with Gitter links as both sides have valuable arguments.
This should be solved for when we start the GSoC-2024 campaign where Gitter is heavily used in the various pages.

WDYT ?

Reply:
Sure @jmMeessen I don't have a very strong opinion about it either way.

Reply:
> I propose to keep the link as it was before.

Makes sense.